src/test/kotlin/io/bazel/kotlin/builder/tasks/js/KotlinBuilderJsTest.java

```diff
@@ -23,6 +23,6 @@ public void testJsErrorRendering() {
                 () ->
                         builder.runCompilationTask(
                                 it -> it.addSource("AClass.kt", "package something", "class AClass{")),
-                lines -> assertThat(lines.get(0)).startsWith(builder.toPlatform("sources/AClass.kt")));
+                lines -> assertThat(lines).contains(builder.toPlatform("sources/AClass.kt:2:14: error: missing '}")));
```

Comment:
There's some additional logging that happens because we aren't using the new JS IR backend. Probably worth making the switch for Javascript compilation in this PR so that we can revert this test back.